<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackJack</class>
 <widget class="QMainWindow" name="BlackJack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlackJack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Cards Left: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_cardsLeft">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Dealer</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="layout_dealer"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Player</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="layout_player"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QToolButton" name="tb_newGame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>new gmae</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_dealHand">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>deal hand</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="tb_shuffleDeck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>shuffle deck</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_hit_me">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>hit me</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_stay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>stay</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>78</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Player:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_playerScore">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Dealer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_dealerScore">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionDeal_Hand"/>
    <addaction name="separator"/>
    <addaction name="actionShuffle_Deck"/>
    <addaction name="actionHit_Me"/>
    <addaction name="actionStay"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewGame">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="toolTip">
    <string>Starts a new game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShuffle_Deck">
   <property name="text">
    <string>Shuffle Deck</string>
   </property>
   <property name="toolTip">
    <string>Shuffles the deck</string>
   </property>
  </action>
  <action name="actionHit_Me">
   <property name="text">
    <string>Hit Me</string>
   </property>
   <property name="toolTip">
    <string>Add another card to your deck</string>
   </property>
  </action>
  <action name="actionStay">
   <property name="text">
    <string>Stay</string>
   </property>
   <property name="toolTip">
    <string>End your turn</string>
   </property>
  </action>
  <action name="actionDeal_Hand">
   <property name="text">
    <string>Deal Hand</string>
   </property>
   <property name="toolTip">
    <string>Deals a new hand</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
